README.md

```diff
@@ -1,15 +1,72 @@
+# nanocar-builder-web
+
 <a href="https://discord.gg/54ySkdv">
   <img src="https://img.shields.io/discord/598249771676860416.svg?logo=discord" alt="chat on Discord">
 </a>
-            
-# nanocar-builder-web
+
 Web based nanocar builder using `three.js`.
 
-## Usage
-- [Download the repository.](https://github.com/formulanano/nanocar-builder-web/archive/master.zip)
-- Unzip the folder.
-- Install project dependencies `npm install`
-- Build for production  `npm run build`
-- Install static file server ` npm install -g serve`
-- Start server `serve -s build`
-- You should see a nanocar molecule in your browser.
+## Getting Started
+
+1- Clone the repo
+
+```bash
+git clone git@github.com:formulanano/nanocar-builder-web.git
+cd nanocar-builder-web
+```
+
+2- Install dependencies
+
+```bash
+npm install
+```
+
+3- Start Run
```

Comment:
At some point we should include instructions for deploying the app to the website. I would like to do it after this pull request but I am not sure how.

Reply:
We should not add these kind of instructions on a public document. It's a private information.
Currently anyone who can download/fork this project can run `npm run build` command to build the app for production and can use its private deploy process to deploy and serve.

For example; it looks like we will choose Netlify for deployment, so we can document privately how we deployed the app using Netlify.